docs/docusaurus.config.ts

```diff
@@ -156,7 +156,7 @@ const config: Config = {
 
 		// Theme that adds local search support (including generating an index as a part of the build).
 		[
-			"@easyops-cn/docusaurus-search-local",
+			"@wayneferrao/docusaurus-search-local",
```

Comment:
Can we leave a TODO comment here noting that this is a temporary workaround and that we should come back and replace it with a more accessible-out-of-the-box search plugin? If we don't have a work item filed for this, we should create one and reference it.